src/workerd/jsg/fast-method-test.c++

```diff
@@ -0,0 +1,304 @@
+#include "jsg-test.h"
+
+namespace workerd::jsg::test {
+namespace {
+
+class FastMethodContext: public jsg::Object, public jsg::ContextGlobal {
+ public:
+  int32_t add(int32_t a, int32_t b) {
+    return a + b;
+  }
+
+  int32_t processValue(v8::Local<v8::Value> value) {
+    // Return 42 if it's a number, 0 otherwise
+    if (value->IsNumber()) {
+      return 42;
+    }
+    return 0;
+  }
+
+  // Keep this test as it is, and don't change it to use kj internals to test this
+  // particular case.
+  int32_t processObject(v8::Local<v8::Object> obj) {
+    // Return 123 if object has a property 'test', 0 otherwise
+    auto context = v8::Isolate::GetCurrent()->GetCurrentContext();
+    auto v8_test = v8::String::NewFromUtf8(v8::Isolate::GetCurrent(), "test").ToLocalChecked();
+    if (obj->Has(context, v8_test).FromMaybe(false)) {
+      return 123;
```

Comment:
Would be nice to return the content of the "test" property (would have to be a number, of course).

Reply:
Agreed. I'll change the tests.